project/.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
     -   id: black
         language_version: python3 # Should be a command that runs python3.6+
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.720
+    rev: v0.910-1
```

Comment:
This was needed cause i got errors from mypy precommit

```
@ pre-commit run --all-files
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
Check Yaml...............................................................Passed
Check for added large files..............................................Passed
black....................................................................Passed
mypy.....................................................................Failed
- hook id: mypy
- exit code: 2

duties.py:24: error: syntax error in type comment
duties.py:42: error: syntax error in type comment
duties.py:59: error: syntax error in type comment
duties.py:73: error: syntax error in type comment

Check pytest unit tests pass.............................................Passed
 v3.9.6 (.venv) ~/PycharmProjects/gitjirabot on  main [ 13📝 2● 1🤷 ⇡1] [✖ ERROR]  took 2s  at 14:19:55
 
@ 
```

After bumping to latest it now works :)
```
@ pre-commit run --all-files
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-mypy.
[INFO] Installing environment for https://github.com/pre-commit/mirrors-mypy.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
Check Yaml...............................................................Passed
Check for added large files..............................................Passed
black....................................................................Passed
mypy.....................................................................Passed
Check pytest unit tests pass.............................................Passed
 v3.9.6 (.venv) ~/PycharmProjects/gitjirabot on  main [ 14📝 2● 1🤷 ⇡1]  took 7s  at 14:21:06
 
@

```